### PR TITLE
[GH-42]Add autocomplete icon

### LINKED ISSUE #42: Add autocomplete icon
Since v5.24 slash commands can provide an `svg` via  `AutocompleteIconData` in `model.Command` that is then shown in the autocomplete list. This ticket is about adding the icon in `assets` to the slash command.

Include screenhots from a dark and a bright theme in the PR description, as there might be the need for a separate icon with a white background for autocomplete.

Similar PRs:
For GitHub plugin https://github.com/mattermost/mattermost-plugin-github/pull/359
For Jira plugin https://github.com/mattermost/mattermost-plugin-jira/pull/653

### PR DESCRIPTION
**Pull request by ByeongsuPark**
#### Summary

Screenshot_2020-10-12_17-03-39
Screenshot_2020-10-12_17-03-58
Add autocomplete icon

#### Ticket Link

  Fixes #42

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,9 +3,22 @@ module github.com/mattermost/mattermost-plugin-jenkins
 go 1.12
 
 require (
+	github.com/RackSec/srslog v0.0.0-20180709174129-a4725f04ec91 // indirect
```
Comment (hanzei) [inline go.mod:6]: Would you please remove the indirect dependencies?

**Review by hanzei — CHANGES_REQUESTED** [review]
LGTM. One small request on `go.mod`

File: go.mod (hunk 1 above)
Comment (ByeongsuPark) [reply go.mod]: Sure. I removed the indirect dependencies and commit it. Would you re-review it?

**Review by hanzei — APPROVED** [review]
Awesome, thanks you :+1:

**Comment by hanzei** [comment]
@ByeongsuPark You need to run `go mod tidy` and commit the result.

**Review by hanzei — APPROVED** [review]

**Review by mickmister — APPROVED** [review]

**Comment by hanzei** [comment]
@DHaussermann This might be a good one for Dev testing

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Icon displays as expected
- Tested on light and dark themes
- Tested on browser and desktop
No issues found 

LGTM! thanks @ByeongsuPark for this enhancment!

**Pull request merged**